src/os/inc/osapi-os-loader.h

```diff
@@ -30,207 +30,18 @@
 #ifndef _osapi_loader_
```

Comment:
Do we want to fix this include guard even though the file is deprecated?

Reply:
> @astrogeco can we move this one forward? It touches everything so the quicker we can get it in the less we are going to have to deal with conflicts.

I'll merge it now and we can revisit my commit in the IC as a hotfix if we want to fix include guard styles for deprecated code.

Reply:
FWIW - I had skipped updating these files because they were deprecated... will probably remove in a future commit with the rest of the deprecated stuff in this cycle.